vyper/venom/passes/pass_manager.py

```diff
@@ -0,0 +1,59 @@
+from typing import Type
+
+from vyper.venom.function import IRFunction
+
+
+class IRPassManager:
```

Comment:
this looks very closely coupled to `IRAnalysis`. maybe it should go in the same file, and also be renamed to `IRAnalysisManager`